src/components/Contact/index.js

```diff
@@ -54,6 +57,44 @@ export default function ContactComponent() {
       toast.error('An error occurred. Please try again.')
     }
   }
+
+  const formConfig = [

```

Comment:
you can pull that outside of the component it has no dependency on the data inside of the component